test/conformance/kernel/kernel_adapter_opencl.match

```diff
@@ -3,3 +3,7 @@ urKernelSetSpecializationConstantsTest.Success/Intel_R__OpenCL___{{.*}}
 urKernelSetSpecializationConstantsTest.InvalidNullHandleKernel/Intel_R__OpenCL___{{.*}}
 urKernelSetSpecializationConstantsTest.InvalidNullPointerSpecConstants/Intel_R__OpenCL___{{.*}}
 urKernelSetSpecializationConstantsTest.InvalidSizeCount/Intel_R__OpenCL___{{.*}}
+urKernelGetSuggestedLocalWorkSizeTest.Success/Intel_R__OpenCL___{{.*}}
```

Comment:
I'm not sure how I feel about adding new failing CTS scenarios. Is this failing with `UR_RESULT_ERROR_UNSUPPORTED_FEATURE` ?

Reply:
Yes. This is because, for Intel OpenCL CPU runtime, the underlying API has been implemented in a version that will release soon, so for now this test will fail. And it should pass after the runtime version uplift to include the version that implemented the API.

Reply:
Thanks, makes sense.

Reply:
I skip those tests if it returns UR_RESULT_ERROR_UNSUPPORTED_FEATURE. Or do you think that I should check for the OpenCL API and skip the whole suite if the OpenCL runtime does not support the API?